network-protection/network-protection-internal/src/main/java/com/duckduckgo/networkprotection/internal/feature/NetPInternalSettingsActivity.kt

```diff
@@ -215,7 +213,7 @@ class NetPInternalSettingsActivity : DuckDuckGoActivity() {
 
         binding.forceRekey.setClickListener {
             lifecycleScope.launch {
-                netPRekeyer.doRekey()
+                sendBroadcast(Intent(DebugRekeyReceiver.ACTION_FORCE_REKEY))
```

Comment:
We use a broadcast as an IPC mechanism, as rekeying needs to be received/executed by the `:vpn` process